ehsm_kms_service/secret_manager_apis.js

```diff
@@ -19,11 +20,14 @@ const {
 } = require('./apis')
 const ehsm_napi = require('./ehsm_napi')
 
-// get a string parameter from payload and base64 if you need
-function getParam_String(payload, key, needBase64) {
-    if (needBase64 == undefined) {
-        needBase64 = true;
-    }
+/**
+ * get a string parameter from payload and base64 if you need
+ * @param {Object} payload : a json object
+ * @param {String} key : the key of payload
+ * @param {boolean} needBase64 [defalut=true] : The return value needs Base64 encoding flag
+ * @returns {String}
+ */
+function getParam_String(payload, key, needBase64 = true) {
```

Comment:
do you plan to refine the check logic in this patch? following the rules of cryptographic_apis

Reply:
no, The public check parameter will be add after this PR is merged